.travis.yml

```
@@ -159,7 +159,9 @@ jobs:
           # Run integration tests.
           version="$(./linkerd version --client --short | tr -cd '[:alnum:]-')"
           ./bin/test-run `pwd`/linkerd linkerd-$version
-
+        - |
+          # Run linkerd-cni integration tests.
+          TAG="$(./linkerd version --client --short)" go test -race ./cni-plugin/test -integration-tests
```

Since it takes this test about to minute to complete, let's add the `-v` flag so that we can see test progress in the logs while the test is still running.

```suggestion
          TAG="$(./linkerd version --client --short)" go test -v -race ./cni-plugin/test -integration-tests
```